content/200-concepts/100-components/02-prisma-client/053-client-extensions/120-query.mdx

```diff
@@ -30,11 +30,18 @@ Use the `$extends` [client-level method](/reference/api-reference/prisma-client-
 
 Use the `query` extension component to modify queries. You can modify a custom query in the following:
 
-- [A specific operation in a specific model](#modify-a-specific-operation-in-a-specific-model)
-- [A specific operation in all models of your schema](#modify-a-specific-operation-in-all-models-of-your-schema)
-- [All operations in a specific model](#modify-all-operations-in-a-specific-model)
-- [All operations in all models of your schema](#modify-all-operations-in-all-models-of-your-schema)
-- [A specific top-level raw query operation](#modify-a-top-level-raw-query-operation)
+- [Enable the preview feature](#enable-the-preview-feature)
+- [Extend Prisma Client query operations](#extend-prisma-client-query-operations)
+  - [Modify a specific operation in a specific model](#modify-a-specific-operation-in-a-specific-model)
+  - [Modify a specific operation in all models of your schema](#modify-a-specific-operation-in-all-models-of-your-schema)
+  - [Modify all operations in a specific model](#modify-all-operations-in-a-specific-model)
+  - [Modify all Prisma Client operations](#modify-all-prisma-client-operations)
+  - [Modify all operations in all models of your schema](#modify-all-operations-in-all-models-of-your-schema)
+  - [Modify a top-level raw query operation](#modify-a-top-level-raw-query-operation)
+  - [Mutate the result of a query](#mutate-the-result-of-a-query)
+- [Wrap a query into a batch transaction](#wrap-a-query-into-a-batch-transaction)
+- [Query extensions versus middlewares](#query-extensions-versus-middlewares)
+  - [If you use the query extension component and middlewares](#if-you-use-the-query-extension-component-and-middlewares)
```

Comment:
This looks auto generated and completely wrong. It should only list the ways how you can modify a client.
The wording of the links is also customized to be _not_ the same as the headlines.

Reply:
Thanks for catching this. It was auto-generated indeed by one of my VS Code extensions. I'll resolve it